editor/plugins/asset_library_editor_plugin.cpp

```diff
@@ -936,70 +936,70 @@ HBoxContainer *EditorAssetLibrary::_make_pages(int p_page, int p_page_count, int
 	if (to > p_page_count)
 		to = p_page_count;
 
-	Color gray = Color(0.65, 0.65, 0.65);
-
 	hbc->add_spacer();
-	hbc->add_constant_override("separation", 10);
+	hbc->add_constant_override("separation", 5);
 
+	Button *first = memnew(Button);
+	first->set_text(TTR("First"));
 	if (p_page != 0) {
-		LinkButton *first = memnew(LinkButton);
-		first->set_text(TTR("first"));
-		first->add_color_override("font_color", gray);
-		first->set_underline_mode(LinkButton::UNDERLINE_MODE_ON_HOVER);
 		first->connect("pressed", this, "_search", varray(0));
-		hbc->add_child(first);
+	} else {
+		first->set_disabled(true);
+		first->set_focus_mode(Control::FOCUS_NONE);
 	}
+	hbc->add_child(first);
 
+	Button *prev = memnew(Button);
+	prev->set_text(TTR("Prev"));
```

Comment:
What about using Previous instead of Prev? This may confuse translators who may not know whether to abbreviate the translated word or not.

Reply:
I think initially it was made, to make everything symmetrical. I agree that `previous` would look more profesional, but not as symmetrical.
![image](https://user-images.githubusercontent.com/27275873/40121425-3783efd8-592a-11e8-997a-ae0fa54f0e08.png)